packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Formik, FastField } from 'formik';
+import { useFormik } from 'formik';
+import { useEffectOnce } from 'react-use';
 import { checkError } from '@caesar/common/utils/formikUtils';
 import {
   AuthTitle,
   AuthDescription,
   MasterPasswordInput,
   Button,
 } from '@caesar/components';
-import { createConfirmPasswordSchema } from './schema';
+import { createConfirmPasswordSchema } from "./schema";
```

Comment:
Don't you use auto-fixer by prettier, do you?

Reply:
Yes, autoimport from WS